torch/csrc/generic/serialization.cpp

```diff
@@ -102,65 +102,65 @@ THWStorage * THPStorage_(readFileRaw)(io file, THWStorage *_storage)
   // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
   int64_t size;
   doRead(file, &size, sizeof(int64_t));
+  int64_t nbytes = element_size * size;
```

Comment:
Why do we multiply this one by element_size here? I don't see the corresponding change in the write code that of a place that used to write the nbytes but now writes a number of element?

Reply:
You're right. Maybe this is the bug!

Reply:
It does seem wrong looking back on it. I'll check it out

Reply:
I'm pretty sure this is actually correct. Line 38 within `writeFileRaw` has:
```
doWrite(fd, &numel, sizeof(int64_t));
```

I also tried changing everything to use `size` instead of `nbytes` below, and I get a ton of errors in `test/test_serialization.py`

Reply:
Yes, and that wasn't changed right?
But all the functions below that were using size (that was and is the number of elements) now use nbytes. I guess that if all functions were updated in this PR to handle it this way that's ok.